oai_agents/agents/rl.py

```diff
@@ -58,6 +58,28 @@ def __init__(self, teammates_collection, args,
                                                                                                    eval_types = eval_types)
         self.best_score, self.best_training_rew = -1, float('-inf')
 
+    @classmethod
+    def generate_randomly_initialized_SP_agent(cls,
+                                               args) -> OAIAgent:
+        '''
+        Generate a randomly initialized learning agent using the RLAgentTrainer class
+        This function does not perform any learning
+
+        :param args: Parsed args object
+        :returns: An untrained, randomly inititalized RL agent
+        '''
+
+        name = 'randomized_agent'
+
+        sp_trainer = cls(name=name,
+                        args=args,
+                        agent=None,
+                        teammates_collection=None,
```

Comment:
This should be {}, because of this [line](https://github.com/HIRO-group/multiHRI/blob/7ebcdaa3097f32bb52f47fede4103b5b1596f33b/oai_agents/agents/rl.py#L112).